Scripts/Golem.gd~RFe296f5.TMP

```diff
@@ -0,0 +1,4 @@
+extends KinematicBody2D
```

Comment:
I don't think this file is needed, shouldn't have been part of the commit. Pro tip. Look at the files that are about to be added before you commit. And if the file looks like it doesn't belong, remove it through git kraken. So remove this file if you can. I'm okay with you just rolling back this commit and applying the changes mentioned in these comments.